web/pano_editor.css

```
@@ -434,6 +423,304 @@
+}
+.pano-video-volume-stack[data-has-audio="true"]:hover .pano-video-volume-pop,
+.pano-video-volume-stack[data-has-audio="true"]:focus-within .pano-video-volume-pop {
+  opacity: 1;
+  visibility: visible;
+  pointer-events: auto;
+  transform: translate(-50%, -12px);
+  transition-delay: 0s;
+}
+.pano-video-volume-slider {
+  width: 88px;
+  height: 16px;
```

Stylelint (17.7.0)

[error] Duplicate property "height" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)